docs/guides/primer/usage_pattern.md

```diff
@@ -166,6 +166,18 @@ index = GPTVectorStoreIndex.from_documents(
 
 See the [Custom LLM's How-To](/how_to/customization/custom_llms.md) for more details.
 
+### Global ServiceContext
+
+If you wanted the service context from the last section to always be the default, you can configure one like so:
+
+```python
+import llama_index
+llama_index.global_serivce_context = service_context
```

Comment:
curious: why do we prefer this variable assignment syntax over say something like `llama_index.set_global_context(service_context)`

Reply:
I personally find the latter more intuitive.

Reply:
fwiw i somewhat agree with simon

Reply:
Sure, easy change!